CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## [2.3.0](https://launchpad.net/mixxx/+milestone/2.3.0)
+## 2.3.1
+* Fix bad phase seek when starting from preroll. [lp1930143](https://bugs.launchpad.net/mixxx/+bug/1930143)  [#4093](https://github.com/mixxxdj/mixxx/pull/4093)
+* AutoDJCratesDAO: Fix crash after loading/unloading tracks. [lp1936396](https://bugs.launchpad.net/mixxx/ [#4107](https://github.com/mixxxdj/mixxx/pull/4107)+bug/1936396)
+
```

Comment:
are you sure this are the only two issues worth mentioning? There have been a bunch of PRs merged to 2.3 since then:
https://github.com/mixxxdj/mixxx/pulls?q=base%3A2.3+merged%3A%3E2021-06-28+is%3Aclosed

Reply:
I was having trouble generating a list of "all PRs since 2.3".  I'll add some more

Reply:
done